docs/en/get_started.md

```diff
@@ -240,7 +240,7 @@ for index, bbox, label_id in zip(indices, bboxes, labels):
 cv2.imwrite('output_detection.png', img)
 ```
 
-You can find more examples from [here](https://github.com/open-mmlab/mmdeploy/demo/python).
+You can find more examples from [here](../../demo/python).
```

Comment:
Let's try this one "https://github.com/open-mmlab/mmdeploy/tree/master/demo/python". Otherwise, this link cannot be accessed in readthedocs